src/gluonts/nursery/sagemaker_sdk/estimator.py

```diff
@@ -474,51 +441,50 @@ def train(
 
         # specify job_name if not set
         if not job_name:
-            job_name = self.__create_job_name(self.base_job_name)
+            job_name = make_job_name(self.base_job_name)
 
+        default_bucket = self.sagemaker_session.default_bucket()
         # needed to set default output and code location properly
         if self.output_path is None:
-            self.output_path = (
-                f"s3://{self.sagemaker_session.default_bucket()}"
-            )
-        sagemaker_log(f"OUTPUT_PATH: {self.output_path}/{job_name}/output")
+            self.output_path = f"s3://{default_bucket}"
+
+        # logger.info(f"OUTPUT_PATH: {self.output_path}/{job_name}/output")
+
         if self.code_location is None:
             code_bucket, _ = parse_s3_url(self.output_path)
             self.code_location = (
-                f"s3://{code_bucket}"  # for consistency with sagemaker API
+                f"s3://{default_bucket}"  # for consistency with sagemaker API
             )
-        sagemaker_log(f"CODE_LOCATION: {self.code_location}/{job_name}/source")
+        logger.info(f"CODE_LOCATION: {self.code_location}/{job_name}/source")
```

Comment:
Actually, this still has to be code_bucket, because the user might choose a different bucket as output then the default one.